numpy/typing/tests/data/reveal/ufuncs.py

```diff
@@ -66,3 +67,14 @@
 reveal_type(np.matmul.identity)  # E: None
 reveal_type(np.matmul(AR_f8, AR_f8))  # E: Any
 reveal_type(np.matmul(AR_f8, AR_f8, axes=[(0, 1), (0, 1), (0, 1)]))  # E: Any
+
+reveal_type(np.bit_count.__name__)  # E: Literal['bit_count']
```

Comment:
@BvB93 I hope this test is right, also would be happy to add other cases for `pass`, `fail`, etc if it makes sense for this case.